.github/workflows/gh-pages.yml

```diff
@@ -16,29 +17,27 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Build the container and the docs inside of it
-        env:
-          TX_TOKEN: ${{ secrets.TX_TOKEN }}
-        run: |
-          docker build . --build-arg tx_token=$TX_TOKEN -t qfield-docs
-          mkdir site
-          docker run -v "${PWD}/site:/opt/app/site" qfield-docs
-          sudo chown -R "${USER}:${USER}" site
-
       - uses: actions/setup-python@v2
         with:
           python-version: 3.9  # transifex-client has compatibility problem with 3.10
 
-      - name: Install requirements
+      - name: Install Python requirements
         run: pip install -r requirements.txt
 
-      - name: Install requirements insiders
+      - name: Install Python requirements insiders
         run: pip install -r requirements-insiders.txt
         env:
           MKDOCS_INSIDERS_TOKEN: ${{ secrets.MKDOCS_INSIDERS_TOKEN }}
 
+      - name: Install Go binary
```

Comment:
Would it make sense to use prepackaged versions instead of building the client in the pipeline?
https://github.com/transifex/cli/releases

Reply:
> Would it make sense to use prepackaged versions instead of building the client in the pipeline? https://github.com/transifex/cli/releases

Absolutely, this is now what we are doing.